# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cargo/hotfix-x86-simd.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cargo-0.88.0/vendor/blake3/build.rs.vanilla	2025-05-29 16:19:52.062654844 +0200
+++ cargo-0.88.0/vendor/blake3/build.rs	2025-05-29 16:23:09.018466895 +0200
@@ -45,7 +45,7 @@
 
 fn is_x86_32() -> bool {
     let arch = &target_components()[0];
-    arch == "i386" || arch == "i586" || arch == "i686"
+    arch == "i386" || arch == "i486" || arch == "i586" || arch == "i686"
 }
 
 fn is_arm() -> bool {
